k8s/build-docker-image-from-source/Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ubuntu:20.04
```

Comment:
In order to avoid large image files, only the necessary environment required by RaftKeeper during execution is retained. What is the purpose of extending `docker/test/integration/base/Dockerfile`?

Reply:
ignore it.